benchmarks/benchmark_config_template.yaml

```diff
@@ -0,0 +1,34 @@
+# Torchserve version is to be installed. It can be one of the options
+#  - branch : "master"
+#  - nightly: "2022.3.16"
+#  - release: "0.5.3"
+# Nightly build will be installed if "ts_version" is not specifiged
+ts_version:
+    branch: &ts_version "master"
+
+# a list of model configure yaml files defined in benchmarks/models_config
+models:
+    - "vgg16.yaml"
+    - "mnist.yaml"
+
+# benchmark on "cpu" or "gpu".
+# "cpu" is set if "hardware" is not specified
+hardware: &hardware "gpu"
+
+# load promethusis metrics report to remote storage if "metrics_cmd" is set
+# the command line to load promethusis metrics report to remote storage
+# Note: keep the values order as the same as the command
+metrics_cmd:
```

Comment:
would be helpful to have a local version of this command as well as a comment - same comment for the cmd below